### ISSUE TITLE
ImageOps.autocontrast typing doesn't permit a tuple value for cutoff, even though docs say it should be allowed

### ISSUE THREAD

**Issue by pa-tna** [python-pillow/Pillow#7785]
### What did you do?

```python
from PIL import Image, ImageOps

img: Image.Image = Image.open("in.png")
img = ImageOps.autocontrast(img, (5, 45))
img = img.convert('1', dither=Image.Dither.NONE)
img.save("out.png")
```

in.png:

<img src="https://github.com/python-pillow/Pillow/assets/60775940/88577960-7bd2-48ad-a0e8-1d984d7c6027" width="260" height="395">

### What did you expect to happen?

`mypy` would validate the above code snippet as containing no typing errors&mdash;the [documentation for `ImageOps.autocontrast`](https://pillow.readthedocs.io/en/stable/reference/ImageOps.html#PIL.ImageOps.autocontrast) says that `cutoff` can be a number or a tuple of length 2.

### What actually happened?

`error: Argument 2 to "autocontrast" has incompatible type "tuple[int, int]"; expected "int"  [arg-type]`

### What are your OS, Python and Pillow versions?

* OS: Windows 10 version 22H2 (OS Build 19045.3930)
* Python: 3.10.11
* Pillow: 10.2.0

**Comment by Yay295** [comment]
`ImageOps.autocontrast` doesn't currently have any typing. It's probably getting it wrong because it has a default value.

https://github.com/python-pillow/Pillow/blob/5bd8fd1dc631f5afde73a25c2ce7b908d9156235/src/PIL/ImageOps.py#L68

**Comment by hugovk** [comment]
Do you have [types-Pillow](https://pypi.org/project/types-Pillow/) installed? 

Typeshed has only `int` there:

https://github.com/python/typeshed/blob/994d22bfbb542956be24580323d5b4aeef822c77/stubs/Pillow/PIL/ImageOps.pyi#L14-L16

**Comment by pa-tna** [comment]
Yes, I do have types-Pillow installed. I can see that Typeshed is the appropriate place to resolve this issue.

It looks like `cutoff` should allow an `int`, `float` or `tuple[int | float, int | float]` value. Does that seem right? If so I can open an issue or PR there instead.

**Comment by radarhere** [comment]
I've created PR #7786 to add a type hint to cutoff of `float | tuple[float, float]`, as https://peps.python.org/pep-0484/#the-numeric-tower
> when an argument is annotated as having type float, an argument of type int is acceptable

However, yes, if you're using types-Pillow, you will need to make a change elsewhere.

**Comment by radarhere** [comment]
You might also be interested to know that the tuple behaviour was added in Pillow 8.0.0 with #4749

**Comment by radarhere** [comment]
https://github.com/python/typeshed/pull/11409 was later merged to change the type hints to allow floats as well.

**Comment by radarhere** [comment]
You might be interested to know that typeshed's Pillow stubs have been [removed](https://github.com/python/typeshed/pull/12732), now that Pillow has it's own type hints.
